### ISSUE TITLE
Scroll on FlatTable create Axe fails - Ensure that scrollable region has keyboard access

### ISSUE THREAD

**Issue by hegou33** [Sage/carbon#4437]
### Current behaviour

We are not able to scroll a FlatTable without select row.

### Expected behaviour

To be able to scroll the FlatTable.
![image](https://user-images.githubusercontent.com/38658070/135271508-10cd1ab1-9a9c-4d6d-8928-5826e127d88f.png)
https://dequeuniversity.com/rules/axe/4.3/scrollable-region-focusable?application=AxeChrome

### CodeSandbox or Storybook URL

https://carbon.sage.com/?path=/story/flat-table--with-sticky-head

### JIRA Ticket (Sage Only)

SBS-20270

### Carbon Version

87.0.1

### What browsers are you seeing the problem on?

Chrome

### What Operating System are you seeing the problem on?

Windows

### Confidentiality

- [X] I confirm there is no confidential or commercially sensitive information included.

**Comment by sjohnson-sage** [comment]
Duplicate of https://github.com/Sage/carbon/issues/4259
